wandb/cli/cli.py

```diff
@@ -1093,17 +1101,20 @@ def launch(
     if (
         uri is None
         and docker_image is None
-        and config.get("uri") is not None
+        and job is None
+        and config.get("uri") is None
+        and config.get("job") is None
```

Comment:
is git also an option?

Reply:
yes it is, I'll get rid of this and put it in the spec validation function.